Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -950,6 +950,9 @@ def identityFS(wf):
                          },
                         step1Defaults])
 
+step1LHEGenDQM = merge([{'-s':'LHE,GEN,VALIDATION:genvalid','--datatier'    : 'GEN-SIM,LHE,DQMIO','--eventcontent': 'LHE,RAWSIM,DQM'},step1GenDefaults])
```

Comment:
Should the datatier and eventcontent be in the same order?

Reply:
See in master, for example,
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L1435

Reply:
I don't think the order matters, as it's this way round in 9_3, which has run fine for a while, but I can change this if it does matter

Reply:
Of course, It matters when you start to use it with relvals. You will register the mismatched dataset between eventcontent and datatier. See output definition from the PR test,
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-947a95/14694/runTheMatrix-results/535.0_TTbar_13TeV_Pow_herwig7+TTbar_13TeV_Pow_herwig7+HARVESTGEN/TT_13TeV_Pow_Herwig7_cff_LHE_GEN_VALIDATION.py

You will end up with `/..../..../LHE` with RAWSIM content and vice versa.

```
# Output definition

process.LHEoutput = cms.OutputModule("PoolOutputModule",
    dataset = cms.untracked.PSet(
        dataTier = cms.untracked.string('GEN-SIM'),
        filterName = cms.untracked.string('')
    ),
    fileName = cms.untracked.string('file:step1.root'),
    outputCommands = process.LHEEventContent.outputCommands,
    splitLevel = cms.untracked.int32(0)
)

process.RAWSIMoutput = cms.OutputModule("PoolOutputModule",
    SelectEvents = cms.untracked.PSet(
        SelectEvents = cms.vstring('generation_step')
    ),
    compressionAlgorithm = cms.untracked.string('LZMA'),
    compressionLevel = cms.untracked.int32(9),
    dataset = cms.untracked.PSet(
        dataTier = cms.untracked.string('LHE'),
        filterName = cms.untracked.string('')
    ),
    eventAutoFlushCompressedSize = cms.untracked.int32(20971520),
    fileName = cms.untracked.string('file:step1_inRAWSIM.root'),
    outputCommands = process.RAWSIMEventContent.outputCommands,
    splitLevel = cms.untracked.int32(0)
)
```

Reply:
OK, thanks for explaining this, I've added another commit to swap the datatier order